### DISCUSSION TITLE
Can't see the video on the page

### DISCUSSION THREAD

**Discussion by maklaud1** [aclap-dev/video-downloadhelper#889] [Help and Q&A]
### Your question

Hello.  DVH stopped seeing videos on the page.  Content subscription site.  About a month ago I could download a video.  Now it says that no video was found on the page.  Everything is updated to the latest versions.  I even tried the beta.  Doesn't see the video. I searched the forum for similar problems, but found nothing.

### Addon & CoApp version and details

Everything is updated to the latest versions. Companion program installed

**Comment by jcfrog** [comment]
Hi,
Probably nothing we can do for the moment but did you report the site?

**Reply by maklaud1** [reply]
I don't know how to report about a site

**Reply by jcfrog** [reply]
oh yes, sorry, the report button appears if the download fails, not in your case which is a detection problem.
Can you tell us which site it is?

**Reply by maklaud1** [reply]
The old version worked on asmrmother.com The current version does not see the video played on the page.  

I would also like to be able to download videos on fansly.  But it doesn't work there either

**Reply by GrampaWildWilly** [reply]
I do not work for VDH.  I'm just another user.  @jcfrog Jérôme, on the other hand, does work for VDH.

I went to asmrmother.com & as expected, I couldn't see anything because you have to join the site.  Fansly, OnlyFans, & a number of other sites have been reported in here as using DRM protection.  Maybe some things are protected & some are not.  Anything that is protected by DRM will not download.  You can search this forum using "drm" as your search key to find more discussion of the topic.

**Reply by maklaud1** [reply]
Of course you need to buy access there.  But I have access.  Perhaps they changed something on the site and installed protection.  It turns out that there is no way to bypass DRM protection and there is no way to download videos from fansley?

**Reply by GrampaWildWilly** [reply]
Correct.  DRM means cannot download.  That is true on any site, not just Fansly.

**Reply by Max345** [reply]
This is still an issue some 9 months later...I subscribe to the same site as OP and VDH still fails to detect the media playing...only sees the first 2mb chunk and then nothing.

**Reply by Mattmjs** [reply]
Yes probably because they still use the same method for the streaming when I looked at it, see my response below and read it.

**Reply by Max345** [reply]
Thanks, yeah I tried that when i originally found this thread...I admittedly know very little about encoding and streaming of files...I guess my question would be is it as simple as modifying VDH to look for different embedded formats or is it more complicated?  I'm surprised more sites arent using whatever streaming format this site does if theyre worried about people downloading streamed content...it had to be a conscious choice, especially since earlier last year it was possible to download content.

**Reply by Mattmjs** [reply]
There are some other sites that I've seen which behave in a similar way. The trick has been to add **.m3u8** to the streaming link.
To which point VDH will detect the media. Of course someone would need to try this out who has access to a video.
I could do this myself , it says $1 to access something but I don't really have a need  to use this site. The other option is to
remotely connect to someone to work out a solution.

**Reply by Max345** [reply]
I'll give it a try later; I assume this is what you are referring to in your post w/ screenshots below?  Is it simply a matter of adding .m3u8 to the end of the url in the hls_manifest?  I'll follow your instructions from the below post and see if i can get anywhere with it...the way VDH works now is that it will see the video playing and offer to download it.  When I do this, it will grab the first 10 seconds or so and thats all...I guess because it's not seeing the other chunks to grab and then stitch together.  I'll try again this evening though with your solution below and see what happens.

**Reply by Max345** [reply]
Haven't had a chance to try yet, but something i found in the FAQs on the mymembersite page...they seem to be actively blocking VDH among other things:
![Screenshot 2025-04-01 at 18-11-45 Enhanced Video Security (DRM) mymember site Help Center](https://github.com/user-attachments/assets/0ea3d97e-4917-412e-8445-1c3b4ac58bcf)

**Comment by Mattmjs** [comment]
You could record the network activity for this site and attach the file here as no one can see the issue. I show how with Firefox.
Go to the video not detected, open the developer tools and make sure nothing is in the network. If there is click the trash can
on the left hand side. Now reload the page, once the page stops loading start playing the video for a few seconds then pause it.
Then do what is shown in the image to save the file.

![3-10-23](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/967c0f2f-483e-4366-8125-b9a91b3ed486)

**Reply by maklaud1** [reply]
Saved the HAR file. What will this give?  Would it be possible to somehow understand how to download?

 The page loads and the video itself starts automatically.  After 2 seconds, I stopped playback and saved the HAR. Upload it here?  Doesn't it contain my data?

**Reply by Mattmjs** [reply]
Yes upload the HAR file here, this is to have some insight on why there is no detection & potentially a fix.
It shouldn't have your data, just all the different elements that make up the page including the video.

**Reply by maklaud1** [reply]
https://cloud.mail.ru/public/ZKen/PQrHjSGoG

**Reply by Mattmjs** [reply]
I see that they use a not so standard format for streaming, no m3u8 or mpd  manifest in the network hence no detection 
in the extension. I can't comment on whether VDH will work for this site ( I'm a user like you ), only the dev would know.
But I found two things of interest, there was something that had stream information such as codecs, frame rates ect..
inside a html file. This could be how the video is streamed. See the screenshot

![Screenshot 2024-06-20 140402](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/2d77f190-5edc-4f43-9e7d-1c14ea8a7764)

The other thing was this json that I found, which had a HLS manifest. Click on the json then Response and scroll down to find it.
If you right click on this and select copy value try pasting it in a new browser tab. Does VDH detect it as a file to download or 
does do something else ?

![Screenshot 2024-06-20 140714](https://github.com/aclap-dev/video-downloadhelper/assets/160297904/08436d93-5b82-4c35-83be-8d6bab8d5bcd)

**Comment by maklaud1** [comment]
mymember.site the video is cached as mp2t but DVH does not detect it

**Reply by Max345** [reply]
I second this, having the same issue all of a sudden with VDH on this site
